content/applications/sales/subscriptions/payment_providers/sdd.rst

```diff
@@ -0,0 +1,214 @@
+**Website**, or **Sales** apps.
+
+For example, navigate to :menuselection:`Accounting app --> Configuration --> Payment Providers`,
+then click the :guilabel:`SEPA Direct Debit` card.
+
+.. tip::
+   The *SEPA Direct Debit* page can also be reached by navigating to :menuselection:`Website app -->
+   Configuration --> Payment Providers`, or :menuselection:`Sales app --> Configuration --> Payment
+   Providers`.
+
+Next, set the :guilabel:`State` to :guilabel:`Enabled` to enable |sdd|. Use the drop-down menus next
+to the :guilabel:`Company` and :guilabel:`Website` fields to set where |sdd| can be used.
+
+The options in the :guilabel:`Configuration` tab can be used to customize how |sdd| is used.
+Optionally tick the checkbox next to :guilabel:`Enable QR Codes` to enable the use of QR codes when
+paying by |sdd|, if desired.
+
+Use the drop-down menu next to the :guilabel:`Communication` field to select the communication type
+used on sales orders. Choose :guilabel:`Based on Document Reference` to use the sales order (or
+invoice number), or :guilabel:`Based on Customer ID` to use the customer identifier. With either
+method selected, communication is given to the customer, once they choose the |sdd| payment method.
+
+The configuration options listed under the :guilabel:`Availability` section are used to set payment
+restrictions, and are optional. Use the :guilabel:`Maximum Amount` field to set a maximum payment
+value, and the :guilabel:`Countries` field to restrict payment from any particular countries, if
+desired.
+
+.. important::
+   The :guilabel:`Currencies` field should be left with the default :guilabel:`EUR` tag to ensure
+   |sdd| is only accessible when customers pay with Euros.
+
+In the :guilabel:`Messages` tab, the :guilabel:`Pending Message` field can be used to add a message
+that will be displayed if the order remains pending after the payment is processed. By default,
+this field contains the default message: :guilabel:`Please make an unique payment transfer to
+confirm the SEPA mandate.`
+
+.. image:: sdd/sdd-payment-provider.png
+   :align: center
+   :alt: The SEPA Direct Debit Payment Provider form.
+
+Once all necessary configuration settings are complete, the :icon:`fa-eye-slash`
+:guilabel:`Unpublished` smart button located at the top of the page automatically changes to
+:icon:`fa-globe` :guilabel:`Published` to show the |sdd| payment option is now visible to customers.
+Only keep the payment method published if payments are ready to be received.
+
+.. tip::
+   It is possible to keep the payment method enabled, but **not** visible to customers, until
+   payments are ready to be received. To do this, set the smart button to :icon:`fa-eye-slash`
+   :guilabel:`Unpublished` after setting the :guilabel:`State` to :guilabel:`Enabled`.
+
+.. note::
+   The |sdd| configuration menu can also be accessed by navigating to :menuselection:`Settings app
+   --> Website --> Shop --> Configure SEPA Direct Debit`.
```

Comment:
I wasn't able to find the settings in the Website section? Perhaps I'm missing some configs?
![image](https://github.com/user-attachments/assets/64cf4102-5a59-4cb8-afbc-5afbc87a5232)

![image](https://github.com/user-attachments/assets/4c78c748-09b5-4287-94f4-c3bc1c19d040)

Reply:
Were you possibly referring to this?

![image](https://github.com/user-attachments/assets/a738f03c-28a4-46bb-8c9e-8d84aee135cb)

If so, the way we'd describe this place using the marketing team's style is:
```suggestion
.. note::
   The |sdd| configuration menu can also be accessed by navigating to :menuselection:`Settings app
   --> Website`, and in the :guilabel:`Shop - Payment` section, click the :guilabel:`Configure SEPA Direct Debit` button.
```